components/gitpod-db/src/workspace-cluster-db.spec.db.ts

```diff
@@ -149,11 +193,20 @@ export class WorkspaceClusterDBSpec {
         };
 
         await this.db.save(wsc1);
+        await this.db.save(wsc1a);
         await this.db.save(wsc2);
 
         const wscs = await this.db.findFiltered({ applicationCluster: "eu02" });
         expect(wscs.length).to.equal(1);
         expect(wscs[0].name).to.equal("eu71");
+
+        const wscs2 = await this.db.findFiltered({ applicationCluster: "us02" });
+        expect(wscs2.length).to.equal(2);
+        wscs2.sort((x, y) => x.name.localeCompare(y.name));
+        expect(wscs2[0].name).to.equal("eu71");
+        expect(wscs2[0].applicationCluster).to.equal("us02");
+        expect(wscs2[1].name).to.equal("us71");
+        expect(wscs2[1].applicationCluster).to.equal("us02");
```

Comment:
Is it not possible to compare lists directly? That would avoid comparing field-by-field which has the risk of still passing when new fields are added.